ansible/roles/prereq/templates/createImmortalDBs.sh.j2

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
we will replace the j2 template with the real ansible way i.e. using modules to generate the immortal DBs